tools/install-dev.sh

```diff
@@ -4,10 +4,10 @@
 
 set -e
 
-if lsb_release -d | grep 'Ubuntu 20.04' > /dev/null; then
-	echo "Ubuntu 20.04 found"
+if lsb_release -d | grep 'Ubuntu 20.04' || lsb_release -d | grep 'Ubuntu 22.04' > /dev/null; then
```

Comment:
**Suggestion:** Consider using a more efficient way to check the Ubuntu version. Instead of using two separate `grep` commands, you can use a single `grep` with an extended regular expression. [performance]
```suggestion
if lsb_release -d | grep -E 'Ubuntu 20.04|Ubuntu 22.04' > /dev/null; then
```